### ISSUE TITLE
[400 BAD REQUEST - InteroperabilityInvalidRequest] Aggiornamento metadata

### ISSUE THREAD

**Issue by apsstn** [ministero-salute/it-fse-support#124]
Buongiorno,

Chiamando l aggiornamento dei metadata riceviamo l`errore in oggetto

Ecco la chiamata

`PUT /govway/rest/in/FSE/gateway/v1/documents/2.16.840.1.113883.2.9.2.120.4.4%5E111111/metadata HTTP/1.1
FSE-JWT-Signature: eyJ4NWMiOiBbIk1JSUZLekNDQXhPZ0F3SUJBZ0lVT0NxMjRSZGFMMWh5clV3Z1ZRc1NRU2NrOHJJd0RRWUpLb1pJaHZjTkFRRUxCUUF3VnpFbk1DVUdBMVVFQXd3ZVEwRWdUV2x1YVhOMFpYSnZJR1JsYkd4aElGTmhiSFYwWlNCVVpYTjBNUjh3SFFZRFZRUUtEQlpOYVc1cGMzUmxjbThnWkdWc2JHRWdVMkZzZFhSbE1Rc3dDUVlEVlFRR0V3SkpWREFlRncweU16QXlNRGt4TXpJeU1qTmFGdzB5TmpBeU1Ea3hNekl5TWpKYU1Ha3hDekFKQmdOVkJBWVRBa2xVTVI4d0hRWURWUVFLREJaTmFXNXBjM1JsY204Z1pHVnNiR0VnVTJGc2RYUmxNUjB3R3dZRFZRUmhEQlJEUmpwSlZDMHdOREl3TURBd01EQXdNREJZV0RFYU1CZ0dBMVVFQXd3UlV6RWpNRFF5TURBd01EQXdNREF3V0Znd2dnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUURNcGc5YjIzK1hHZFJmbUNTMXdYdkFOVnZncUdtU2laNzB5Q3QrM2hBbWY5VG1kbkZWMUVhSEFBOVZkTDB3bmhlQXVSZWpvaXlwcHdScjR5R2EzRHlLM1doYjdwZUhkVjRNNGdScnNtWnR4ZFljdHliNmRuSDhDUUVVM1NGbm9tYWRpemw5WTdHVzZLVkdwOUdkQm82WWw0RnBWTUtwQU0zcDVDMG9SZWxZMWVnMW1iTlhDK29mL1o2ZGgwK0g3MDJaK2toMnY0WlQ4V2k2TDRjK2JlOHhRT0RSWSt5Und3V1JLOXpiTHQvZXBMWWhycUN6ZkNORmVLeUZxUnJyd042NDBmQnNxYXZxMWxQNE9qUmJaaEtMdEd4Wkh6TzkyaGpvRGo3ZEFEZFI5MzFBcFVXTExTMHdkdmo5SEROajZPenhha1BYcVZXa29MTHdrVEhzZEVXRkFnTUJBQUdqZ2R3d2dka3dId1lEVlIwakJCZ3dGb0FVRmptcEdnV0p0dzh0c2pYdHdnbWF4UU9ESWo4d0VnWURWUjBnQkFzd0NUQUhCZ1VyVEIwQkN6QnpCZ05WSFI4RWJEQnFNR2lnWnFCa2htSnNaR0Z3T2k4dlkyRmtjeTVtYVc1aGJucGxMbWwwTDBOT1BVTkJKVEl3VFdsdWFYTjBaWEp2SlRJd1pHVnNiR0VsTWpCVFlXeDFkR1VsTWpCVVpYTjBMRTg5VFdsdWFYTjBaWEp2SlRJd1pHVnNiR0VsTWpCVFlXeDFkR1VzUXoxSlZEQWRCZ05WSFE0RUZnUVVSVnZjYjlyNVB4Zk9JV0hsdHk4UkVoZlN2Rmt3RGdZRFZSMFBBUUgvQkFRREFnWkFNQTBHQ1NxR1NJYjNEUUVCQ3dVQUE0SUNBUUFacmZjWEEyaUtYWUFlSDM1d1kvOHlJcGtlRTl2OHJseUV4MCs0aEpmbEdwQzBOa3ZuWm9aVjJmTGpRbldhek1pUHpVekd4dXdYN1pNVytXMzAzTWM0bGxZYWJGbmR6RlpFNG9JcWZZQ1MyOTlBalVpU2c1OEowMjVjNUhGOEVwZUZ0TlJnRStQODJpYU92eFdGU296bXY1bEdOTUhQMHBCS2duZmVSM3V4dUptYWxPK0U3cWhvZkZnQ3E3cHp2SGwrUnJvM0ZqSWRuT2FyWTVPamI5SURHblgzMmh4a0hxUlBKQ29zSjVvbVdwUEpFMEZoaUo0MEUyd3BnbkxNdHJHUk5oWTNrOVZpeXVOazJlWkQ1alZFVXorSkdGQ3BRcEE2MG96RlZ5K3oxZ2FVNjhCNUpLb0JuSUtHTWRwVEpoN1I1NC9DWCtlaFNSTEhXZzV5K0JmaFdJVzdQUForZGR1UFpsY2FJY3ZkSU1KRUlhZFVjb3JiSmxkdWZKOUxWL2FmUDFWL1BaOXlINnk5MERLQ2VMdXQ1QTdvV3pzZDdIdnRONEpBaXVvRW1FY3pHQ010UzJYZWNFVExXNFdKV0g0dkdhZkNTb1Bac1ZYLzM4TXZCem9WRWNVKzNrbmxFeHd4OWlNOE5rQmxjNjhrL1RMS3hlOWNodkxZQzY5L3psWFV1bWtiMVNxTmFlTmtwZjdRay85RU8zWTNMenp1UEJmK2ZIRFFzdWV4c2E5TWx5MEVmdFRXRk1rVG9IVWJ5bDAyN3E0NldpZXBGL1hHa1NzQ2hERnZPMy9WdUsrelRYUzNMaE9IZTJlY296LzVyZDNFelI5dGdzZXVQWFBNTHVXc2M5YVBsVGV5eGVBWnFZSHZ0aWNWTE9sSkk0NlczbW53WHlQYU5uaFhNZz09Il0sImFsZyI6ICJSUzI1NiIsInR5cCI6ICJKV1QifQ.eyJpc3MiOiAiaW50ZWdyaXR5OlMxIzA0MjAwMDAwMDAwMFhYIiwiaWF0IjogMTY3NzQ5NDYxMCwiZXhwIjogMTY3NzQ5ODIxMCwianRpIjogIjQ3YmUzN2I4LWRhNDItNGQzNy05ZjZmLTFjNGYxODhiYTZiYSIsImF1ZCI6ICJodHRwczovL21vZGlwYS12YWwuZnNlLnNhbHV0ZS5nb3YuaXQvZ292d2F5L3Jlc3QvaW4vRlNFL2dhdGV3YXkvdjEiLCJzdWIiOiAiU1NTTU5ONzVCMDFGMjU3TF5eXiYyLjE2Ljg0MC4xLjExMzg4My4yLjkuNC4zLjImSVNPIiwic3ViamVjdF9vcmdhbml6YXRpb25faWQiOiAiMTkwIiwic3ViamVjdF9vcmdhbml6YXRpb24iOiAiUmVnaW9uZSBTaWNpbGlhIiwibG9jYWxpdHkiOiAiMjAxMTIzNDU2IOKAkyAyMDEiLCJzdWJqZWN0X3JvbGUiOiAiQUFTIiwicGVyc29uX2lkIjogIkdUV0dXWTgyQjQyRzkyME1eXl4mMi4xNi44NDAuMS4xMTM4ODMuMi45LjQuMy4yJklTTyIsInBhdGllbnRfY29uc2VudCI6IHRydWUsInB1cnBvc2Vfb2ZfdXNlIjogIlVQREFURSIsInJlc291cmNlX2hsN190eXBlIjogIignMTE1MDItMl5eMi4xNi44NDAuMS4xMTM4ODMuNi4xJykiLCJhY3Rpb25faWQiOiAiVVBEQVRFIiwic3ViamVjdF9hcHBsaWNhdGlvbl9pZCI6ICIxMjAiLCJzdWJqZWN0X2FwcGxpY2F0aW9uX3ZlbmRvciI6ICJSZWdpb25lIExhemlvIiwic3ViamVjdF9hcHBsaWNhdGlvbl92ZXJzaW9uIjogIlIyIn0.ZhK5MIWIwNyasLg6vv10tiuU7sX-HOmpcrSMs8-nsWu_D-j96_7ANCBktG6svGbfa6s61lNULKR0HOrdNCAdBCwtXKRhNvlZUSMBy6-Ufzo5hAIdi9SvR0UyHu42Ntdrp8Aa_DWK_FZK9aAV1To3j4h7H5UzPUUMjuYFt7Sw7mlB_c07RiuZf_RXyefFjCDHpp4poLI7NH7D0lLR0nIsrmX_hxTrlxOY2JwJLCtESJWrpaCwPLV60sqU-XiKwF1c_eKiIHNSnrDfD2ALcqPhmd4fWe9QlTuvv1-6qmJ5DvII7Xi_-PBDMjHszl0Pc54S-KXq1O80JvzQ2gWy763kBA
Authorization: eyJ4NWMiOiBbIk1JSUZLekNDQXhPZ0F3SUJBZ0lVT0NxMjRSZGFMMWh5clV3Z1ZRc1NRU2NrOHJJd0RRWUpLb1pJaHZjTkFRRUxCUUF3VnpFbk1DVUdBMVVFQXd3ZVEwRWdUV2x1YVhOMFpYSnZJR1JsYkd4aElGTmhiSFYwWlNCVVpYTjBNUjh3SFFZRFZRUUtEQlpOYVc1cGMzUmxjbThnWkdWc2JHRWdVMkZzZFhSbE1Rc3dDUVlEVlFRR0V3SkpWREFlRncweU16QXlNRGt4TXpJeU1qTmFGdzB5TmpBeU1Ea3hNekl5TWpKYU1Ha3hDekFKQmdOVkJBWVRBa2xVTVI4d0hRWURWUVFLREJaTmFXNXBjM1JsY204Z1pHVnNiR0VnVTJGc2RYUmxNUjB3R3dZRFZRUmhEQlJEUmpwSlZDMHdOREl3TURBd01EQXdNREJZV0RFYU1CZ0dBMVVFQXd3UlV6RWpNRFF5TURBd01EQXdNREF3V0Znd2dnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUURNcGc5YjIzK1hHZFJmbUNTMXdYdkFOVnZncUdtU2laNzB5Q3QrM2hBbWY5VG1kbkZWMUVhSEFBOVZkTDB3bmhlQXVSZWpvaXlwcHdScjR5R2EzRHlLM1doYjdwZUhkVjRNNGdScnNtWnR4ZFljdHliNmRuSDhDUUVVM1NGbm9tYWRpemw5WTdHVzZLVkdwOUdkQm82WWw0RnBWTUtwQU0zcDVDMG9SZWxZMWVnMW1iTlhDK29mL1o2ZGgwK0g3MDJaK2toMnY0WlQ4V2k2TDRjK2JlOHhRT0RSWSt5Und3V1JLOXpiTHQvZXBMWWhycUN6ZkNORmVLeUZxUnJyd042NDBmQnNxYXZxMWxQNE9qUmJaaEtMdEd4Wkh6TzkyaGpvRGo3ZEFEZFI5MzFBcFVXTExTMHdkdmo5SEROajZPenhha1BYcVZXa29MTHdrVEhzZEVXRkFnTUJBQUdqZ2R3d2dka3dId1lEVlIwakJCZ3dGb0FVRmptcEdnV0p0dzh0c2pYdHdnbWF4UU9ESWo4d0VnWURWUjBnQkFzd0NUQUhCZ1VyVEIwQkN6QnpCZ05WSFI4RWJEQnFNR2lnWnFCa2htSnNaR0Z3T2k4dlkyRmtjeTVtYVc1aGJucGxMbWwwTDBOT1BVTkJKVEl3VFdsdWFYTjBaWEp2SlRJd1pHVnNiR0VsTWpCVFlXeDFkR1VsTWpCVVpYTjBMRTg5VFdsdWFYTjBaWEp2SlRJd1pHVnNiR0VsTWpCVFlXeDFkR1VzUXoxSlZEQWRCZ05WSFE0RUZnUVVSVnZjYjlyNVB4Zk9JV0hsdHk4UkVoZlN2Rmt3RGdZRFZSMFBBUUgvQkFRREFnWkFNQTBHQ1NxR1NJYjNEUUVCQ3dVQUE0SUNBUUFacmZjWEEyaUtYWUFlSDM1d1kvOHlJcGtlRTl2OHJseUV4MCs0aEpmbEdwQzBOa3ZuWm9aVjJmTGpRbldhek1pUHpVekd4dXdYN1pNVytXMzAzTWM0bGxZYWJGbmR6RlpFNG9JcWZZQ1MyOTlBalVpU2c1OEowMjVjNUhGOEVwZUZ0TlJnRStQODJpYU92eFdGU296bXY1bEdOTUhQMHBCS2duZmVSM3V4dUptYWxPK0U3cWhvZkZnQ3E3cHp2SGwrUnJvM0ZqSWRuT2FyWTVPamI5SURHblgzMmh4a0hxUlBKQ29zSjVvbVdwUEpFMEZoaUo0MEUyd3BnbkxNdHJHUk5oWTNrOVZpeXVOazJlWkQ1alZFVXorSkdGQ3BRcEE2MG96RlZ5K3oxZ2FVNjhCNUpLb0JuSUtHTWRwVEpoN1I1NC9DWCtlaFNSTEhXZzV5K0JmaFdJVzdQUForZGR1UFpsY2FJY3ZkSU1KRUlhZFVjb3JiSmxkdWZKOUxWL2FmUDFWL1BaOXlINnk5MERLQ2VMdXQ1QTdvV3pzZDdIdnRONEpBaXVvRW1FY3pHQ010UzJYZWNFVExXNFdKV0g0dkdhZkNTb1Bac1ZYLzM4TXZCem9WRWNVKzNrbmxFeHd4OWlNOE5rQmxjNjhrL1RMS3hlOWNodkxZQzY5L3psWFV1bWtiMVNxTmFlTmtwZjdRay85RU8zWTNMenp1UEJmK2ZIRFFzdWV4c2E5TWx5MEVmdFRXRk1rVG9IVWJ5bDAyN3E0NldpZXBGL1hHa1NzQ2hERnZPMy9WdUsrelRYUzNMaE9IZTJlY296LzVyZDNFelI5dGdzZXVQWFBNTHVXc2M5YVBsVGV5eGVBWnFZSHZ0aWNWTE9sSkk0NlczbW53WHlQYU5uaFhNZz09Il0sImFsZyI6ICJSUzI1NiIsInR5cCI6ICJKV1QifQ.eyJpc3MiOiAiYXV0aDpTMSMwNDIwMDAwMDAwMDBYWCIsImlhdCI6IDE2Nzc0OTQ2MTAsImV4cCI6IDE2Nzc0OTgyMTAsImp0aSI6ICI3ZGVmMzQzMC0yY2FmLTRhOGUtOGRlMS1hZGM1MjUxZDQxYmQiLCJhdWQiOiAiaHR0cHM6Ly9tb2RpcGEtdmFsLmZzZS5zYWx1dGUuZ292Lml0L2dvdndheS9yZXN0L2luL0ZTRS9nYXRld2F5L3YxIiwic3ViIjogIlNTU01OTjc1QjAxRjI1N0xeXl4mMi4xNi44NDAuMS4xMTM4ODMuMi45LjQuMy4yJklTTyJ9.xuYOCH6E_xT9CsAizl2g9t1es_ZV-q0fupNR8bM27GFcE9mt359YyYDYLODd39Ww5U0NgdsOetOmWG7xb-OAPn0wlP8rQQyJ47zrj35SJZH27qp-G-Wy7Dno3HstIeYB-gfpFxXSkC3zE-_0fr8U76HzjMJa2_S2o-gekOd0JzdCI-Mq4Kb08Wo8rPL7fS1reP7-JpR9OyAsZ0JvNpC0RrgNhFi6T5njI8ri63MNk-Qxjuh7Hv8nWeSFk8UGoAiM-DKE3BqOuZzTdKSDFD5JfsICSBlXu6zeXue5kdFxo57MOcqVxFxyVXHZCl7fQS8XqmPKsuN6whYOyYRevWzGnQ
x-correlation-id: 932861c1-b68b-11ed-9a88-bcd0746b7ce1
Host: modipa-val.fse.salute.gov.it:443
User-Agent: AHC/1.0
Connection: keep-alive
Accept: */*
Content-Type: application/json; charset=UTF-8
Content-Length: 380

{
  "tipologiaStruttura": "Ospedale",
  "attiCliniciRegoleAccesso": [
    "PUBLICPOL"
  ],
  "tipoDocumentoLivAlto": "WOR",
  "assettoOrganizzativo": "AD_PSC001",
  "dataInizioPrestazione": "20141020110012",
  "dataFinePrestazione": "20141020110012",
  "conservazioneANorma": "CIAO",
  "tipoAttivitaClinica": "PHR",
  "identificativoSottomissione": "2.16.840.1.113883.2.9.4.3.2"
}`

Ecco la risposta

`HTTP/1.1 400 Bad Request
X-Backside-Transport: FAIL FAIL
Connection: Keep-Alive
Transfer-Encoding: chunked
Expires: 0
Cache-Control: no-cache, no-store, must-revalidate
Server: GovWay-SOGEI
Pragma: no-cache
GovWay-Message-ID: caab73b8-b686-11ed-a9a6-005056ae7395
GovWay-Transaction-ErrorType: InteroperabilityInvalidRequest
GovWay-Transaction-ID: c80cd406-b686-11ed-a9a6-005056ae7395
Date: Mon, 27 Feb 2023 10:43:38 GMT
Vary: *
X-Content-Type-Options: nosniff
Content-Type: application/problem+json
X-Global-Transaction-ID: 3f7e3e2d63fc895a4ae91b8d

DEBUG 2023-02-27 11:43:38,764 [[gtw-fse-s-api].http.requester.HTTP_gtw_configuration.01 SelectorRunner] [processor: implementations.request/processors/0; event: 932861c1-b68b-11ed-9a88-bcd0746b7ce1] org.mule.service.http.impl.service.HttpMessageLogger.http.requester.HTTP_gtw_configuration: REQUESTER
112
{"type":"https://govway.org/handling-errors/400/InteroperabilityInvalidRequest.html","title":"InteroperabilityInvalidRequest","status":400,"detail":"Received request is not conform to the required interoperability profile","govway_id":"c80cd406-b686-11ed-a9a6-005056ae7395"}
`

**Comment by masciamariotti** [comment]
Buongiorno,
da una prima analisi sembrerebbe che non stia inviando "Bearer" nel parametro header "Authorization"
es:
Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5c ... iZPqKv3kUbn1qzLg

potrebbe verificare e riprovare?
Grazie
